task-management/src/main/webapp/skin/css/tasks.less

```diff
@@ -3013,6 +3013,7 @@
     }
     .changesText {
       max-width: 270px;
+      padding-bottom: 1px;
```

Comment:
please use a vue css helper

Reply:
I can't use vue CSS helper in this case because the increment of the property (margin and padding ) in 4px intervals